images/build/Dockerfiles/vso.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM oryxdevmcr.azurecr.io/public/oryx/build AS main
+ARG SDK_STORAGE_ENV_NAME
+ARG SDK_STORAGE_BASE_URL_VALUE
+ARG AI_KEY
+ARG GIT_COMMIT=unspecified
+ARG BUILD_NUMBER=unspecified
+ARG RELEASE_TAG_NAME=unspecified
+
+RUN rm -rf /tmp/oryx
+
+ENV PATH="$ORIGINAL_PATH:/opt/oryx:/opt/nodejs/lts/bin:/opt/dotnet/sdks/lts:/opt/python/latest/bin:/opt/yarn/stable/bin:/opt/hugo"
```

Comment:
nit: may be append $original_path at end like earlier?

Reply:
No, we should not be doing that here. Please take a look at my earlier comment.